src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -178,6 +184,15 @@
     <Compile Include="$(CommonPath)\Interop\Unix\libssl\Interop.SafeSslHandle.cs">
       <Link>Interop\Unix\libssl\Interop.SafeSslHandle.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libssl\SecuritySafeHandles.cs">
+      <Link>Interop\Unix\libssl\SecuritySafeHandles.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libssl\StreamSizes.cs">
+      <Link>Interop\Unix\libssl\StreamSizes.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libssl\SslConnectionInfo.cs">
+      <Link>Interop\Unix\libssl\SslConnectionInfo.cs</Link>
```

Comment:
Note that these were deliberately not put into Interop\libssl earlier because they are specific to System.Net.Security.SslStream. There is a danger that due to dev carelessness, the non-Security specific code might start getting mixed with this (eg. adding mapping code in Interop.Openssl.cs). Will leave it to you if you still want to organize code like this.